lib/project.js

```diff
@@ -102,7 +118,7 @@ function Project$get(key, dflt) {
       break;
   }
   return _.get(this.normalizedPkgJSON, key,
-            _.get(this.rawPkgJSON, key, dflt));
+               _.get(this.rawPkgJSON, key, dflt));
```

Comment:
I heartily disklike coding styles where the indent of one line depends on the length of the function names in the previous. obviously, not a blocking concern!

Reply:
A valid concern. It creates extra diff noise when things get renamed.

I fiddled with the rules to reduce the diff, but didn't try very hard. I can take another pass to see if there's a better config value.